lightning-tag/faq.md

```diff
@@ -71,6 +71,21 @@ Make sure that the onProfileReady() event is fired before rendering your ads. Th
   </script>
   
   <script async src="https://tags.crwdcntrl.net/lt/c/<lotameClientId>/lt.min.js"></script>
-</head>```
+</head>
+```
 
 If you still need assistance, please reach out to Lotame by emailing support@lotame.com.
+
+## How can I access Lotame's profile id in my browser?
```

Comment:
This looks good. Can we reference this in the Google Ad Manager page somewhere? 

"You may also need to push the Lotame profile Id into the Google Ad Manager call (ex. this is required before you can send Lotame your Google Ad Log files for automated campaign analytics). If you need to pass the Lotame profile Id into the Google Ad Manager ad calls, please reference <link>"

Reply:
Once I get my other prs merged in, I will update that page (since i have other changes on there already, it is easier to wait till those are merged).